### DISCUSSION TITLE
Viewing all the notifications

### DISCUSSION THREAD

**Discussion by VipinDhakad** [leandrosimoes/react-native-android-notification-listener#63] [Q&A]
What this module or example you provided is currently doing is, it stores a notification as lastNotification in the AsynStorage and render it but I want to store all the notifications that are coming to device and render them. I tried using an array state for the storage and push the incoming notification into that array and store the new data into the AsynStorage, but it is not working.

Any help?
![github](https://user-images.githubusercontent.com/84377420/225348360-ac51857d-6e77-4670-9390-b9c63ec7fe3d.png)

**Comment by leandrosimoes** [comment]
I would need more information to try to check what is happening.

* There are any errors on the console?
* What is the result of the `getItem`?

**Comment by thanhhieu12** [comment]
hi ad , I got an error when I couldn't read the notification when I reopened the app . 
Looking forward to your help
thanks
